src/components/BingoMachine/NumberDisplay/index.tsx

```diff
@@ -0,0 +1,68 @@
+import styled, { keyframes } from "styled-components";
+import { DISPLAY_NUMBER_INITIAL } from "../../BingoMachine";
+
+type Props = {
+  displayNumber: number;
+  extractedNumbers: number[];
+}
+
+const NumberDisplay = ({
+  displayNumber,
+  extractedNumbers
+}: Props) => {
+
+  const content = (() => {
+
+    if (displayNumber !== DISPLAY_NUMBER_INITIAL) {
+      return displayNumber;
+    }
+
+    if (extractedNumbers.length !== 0) {
+      return extractedNumbers[extractedNumbers.length - 1];
```

Comment:
ここはこういう書き方もできます。(https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/at#%E8%A7%A3%E8%AA%AC)
```suggestion
      return extractedNumbers.at(-1);
```

Reply:
知りませんでした。
ありがとうございます！